### PR TITLE
fix tests

### LINKED ISSUE #25: CircleCI doesn't fail despite tests failing
Make sure the change doesn't break any existing pages by adding a link checker to the CircleCI.

This possibly should be optional at first until we fix all currently broken links.

### PR DESCRIPTION
**Pull request by butlerx**
close #25

### CODE REVIEW

**Review by VoyTechnology — APPROVED** [review]

**Comment by VoyTechnology** [comment]
Output from https://circleci.com/gh/redbrick/docs/105
```
Creating docs-test
Attaching to docs-test
docs-test     | Running ["ScriptCheck", "LinkCheck", "ImageCheck"] on ["site"] on *.html... 
docs-test     | 
docs-test     | 
docs-test     | Checking 0 external links...
docs-test     | Ran on 0 files!
docs-test     | 
docs-test     | 
docs-test     | HTML-Proofer finished successfully.
docs-test exited with code 0
```

**Comment by butlerx** [comment]
fail is in build 
```
docs-build    | 
docs-build    | Config file '/usr/src/app/mkdocs.yml' does not exist.
docs-build exited with code 1
```

**Comment by butlerx** [comment]
the tests are now working but its stilling getting a pass despite being errors

**Review by VoyTechnology — COMMENTED** [review]
Is this ready to go now?

**Comment by butlerx** [comment]
no 
```
docs-test     | Running ["HtmlCheck", "ScriptCheck", "LinkCheck", "ImageCheck"] on ["site"] on *.html... 
docs-test     | 
docs-test     | 
docs-test     | Ran on 152 files!
docs-test     | 
docs-test     | 
docs-test     | - site/machines/paphos/index.html
docs-test     |   *  internally linking to ../../procedures/newboards, which does not exist (line 823)
docs-test     |      <a href="../../procedures/newboards">boards</a>
docs-test     |   *  internally linking to ../services/dovecot, which does not exist (line 815)
docs-test     |      <a href="../services/dovecot">dovecoat</a>
docs-test     | htmlproofer 3.6.0 | Error:  HTML-Proofer found 2 failures!
docs-test exited with code 1
```
it should be failing

**Comment by VoyTechnology** [comment]
I can fix those links

**Comment by butlerx** [comment]
yes but how is it getting a pass if the test failed

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,4 +18,6 @@ jobs:
           command: docker-compose up build-docs
       - run:
           name: Test Internal Links
-          command: docker-compose up test
+          command: |
+            set -x
```
Comment (VoyTechnology) [inline .circleci/config.yml]: `set -e`?

**Comment by VoyTechnology** [comment]
Agreed with @butlerx to divide this PR into 2, this one will be used to report the errors, another one will actually make the tests fail

**Pull request merged**